packages/core/src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -38,15 +45,30 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
             throw new InvalidArgumentError(`Unable to initialize payment because "options.braintreepaypal" argument is not provided.`);
         }
 
-        const state = await this._store.dispatch(this._checkoutActionCreator.loadDefaultCheckout());
-        const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
-        const currency = state.cart.getCartOrThrow()?.currency.code;
+        let state: InternalCheckoutSelectors;
+        let paymentMethod: PaymentMethod<any>;
```

Comment:
Can we avoid the use of `any`?

Reply:
Yes
It is default generic type for PaymentMethod
I removed it, thanks